pkg/controllers/subnet/namespace_handler.go

```diff
@@ -18,28 +18,28 @@ import (
 	"github.com/vmware-tanzu/nsx-operator/pkg/apis/vpc/v1alpha1"
 )
 
-// Subnet controller should watch event of namespace, when there are some updates of namespace labels,
+// Subnet controller should watch event of Namespace, when there are some updates of Namespace labels,
 // controller should build tags and update VpcSubnet according to new labels.
 
 type EnqueueRequestForNamespace struct {
 	Client client.Client
 }
 
 func (e *EnqueueRequestForNamespace) Create(_ context.Context, _ event.CreateEvent, _ workqueue.RateLimitingInterface) {
-	log.V(1).Info("namespace create event, do nothing")
+	log.V(1).Info("Namespace create event, do nothing")
```

Comment:
If I remember correctly, we use lower case for log message

Reply:
Why do we persist in following the wrong path until it's too late? It's better to make changes as early as possible. If we adjust a little bit with each patch, we will be in the right way sooner or later. It's not too late to mend the fold after the sheep are lost.